birdhouse/default.env

```diff
@@ -11,6 +11,11 @@ export BASH_IMAGE="bash:5.1.4"
 # Root directory under which all data persistence should be nested under
 export DATA_PERSIST_ROOT="/data"
 
+# shellcheck disable=SC2016
+# Root directory for all files that are persisted on disk and may contain links (ie. the files
+# are "shared" between subdirectories). This means that the subdirectory structure is fixed.
```

Comment:
I think I just meant the files in this directory should not be moved once created so that the links don't break. But I agree that the wording is not great. I'll update it.

Reply:
On Weaver front, it should be valid as well since WPS outputs dir remains mounted the same way on both host/docker side.